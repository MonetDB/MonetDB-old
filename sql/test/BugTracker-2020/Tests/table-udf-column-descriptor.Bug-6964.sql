START TRANSACTION;
CREATE OR REPLACE FUNCTION test(i integer) 
RETURNS TABLE (value integer)
BEGIN
  return select value from generate_series(1,i);
END;

select value from test(3);
ROLLBACK;
