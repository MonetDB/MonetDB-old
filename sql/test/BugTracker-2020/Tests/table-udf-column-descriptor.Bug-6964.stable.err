stderr of test 'table-udf-column-descriptor.Bug-6964` in directory 'sql/test/BugTracker-2020` itself:


# 12:20:49 >  
# 12:20:49 >  "mclient" "-lsql" "-ftest" "-tnone" "-Eutf-8" "-i" "-e" "--host=/var/tmp/mtest-261339" "--port=32116"
# 12:20:49 >  


# 12:20:49 >  
# 12:20:49 >  "Done."
# 12:20:49 >  

