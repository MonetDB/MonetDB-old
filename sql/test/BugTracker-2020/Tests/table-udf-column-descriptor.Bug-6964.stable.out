stdout of test 'table-udf-column-descriptor.Bug-6964` in directory 'sql/test/BugTracker-2020` itself:


# 12:20:49 >  
# 12:20:49 >  "mclient" "-lsql" "-ftest" "-tnone" "-Eutf-8" "-i" "-e" "--host=/var/tmp/mtest-261339" "--port=32116"
# 12:20:49 >  

#START TRANSACTION;
#CREATE OR REPLACE FUNCTION test(i integer) 
#RETURNS TABLE (value integer)
#BEGIN
#  return select value from generate_series(1,i);
#END;
#select value from test(3);
% .%2 # table_name
% value # name
% int # type
% 1 # length
[ 1	]
[ 2	]
#ROLLBACK;

# 12:20:49 >  
# 12:20:49 >  "Done."
# 12:20:49 >  

